pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -23,25 +24,27 @@ func (e *cloudWatchExecutor) parseResponse(metricDataOutputs []*cloudwatch.GetMe
 		}
 
 		for _, r := range mdo.MetricDataResults {
```

Comment:
I would insert something like:

```
id := *r.id
lablel := *r.Label
```

as the start of the inside of the loop. That should make it easier to read, and also if you need to do a nil check to make sure you are not dereferencing a nil pointer -  that would be the place to do it.

Reply:
Thanks @kylebrandt, does make sense indeed and I'll take care of it.